index.html

```diff
@@ -57,141 +57,51 @@ <h3 class="connect">LET'S CONNECT</h3>
           </ul>
         </div>
       </div>
-    </section>
-
-    <section id="Portfolio" class="works">
 
-      <div class="projects" >
-        <div>
-          <img class="mobile1" src="./images/Snapshoot Portfolio.png" alt="Mobile1">
-          <img class="desktop1" src="./images/desktop1.png" alt="">
-        </div>
-        <div class="Project-div2">
-          <h2 class="works-title">TONIC</h2>
-          <p><span class="canopy">Canopy</span> <img class="Bullet-point" src="./images/Counter.png" alt="Counter">
-            <span class="dev"> Back End Dev
-              <img class="Bullet-point" src="./images/Counter.png" alt="Counter"> 2015</span>
-          </p>
-          <div class="project-description">
-            <p class="white-background para">A daily selection of privately personalized reads; no accounts or sign-ups
-              required.</p>
+</section>
+<section class="pop-up">
+  <div class="inner-pop-up">
+    <div class="popCloseLayout">
+    <h1 class="works-title">TONIC</h1>
+    <div id="closelink">
+    <p class="popup-close">X</p>
+    </div>
+    </div>
+    <p><span class="canopy">Canopy</span> <img class="Bullet-point" src="./images/Counter.png" alt="Counter">
+                <span class="dev"> Back End Dev
+                  <img class="Bullet-point" src="./images/Counter.png" alt="Counter"> 2015</span>
+              </p>
+              <img class="popimg popMobImg" src="./images/popupMobile.png" alt="">
+              <img class="popimg popDeskImg" src="./images/DesktopPop.png" alt="">
+              <div class="layout-pop">
+              <div>
+              <p class="Pop-desktop-p">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+              <br />
+              <br />
+              Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.</p>

```

Comment:
- [ ] The image for each popup should be dynamically displayed just like every other content. Currently what you have is a single image showing for all project, however each project has its own image and this should be what is shown once the user clicks to see the project. please kindly make this desired change so that your popup implementation can be complete. you have done a great job already :+1: :sparkles: 
- [ ] Your projects array holds all of the information for each project and it is based on the contents of each project object that the popup is to be displayed. you are not required to hardcord your popup like you have done in this highlighted html code.
- > see this project's example image below
- Expected pop-up
![ep](https://github.com/MargaretTechworld/Portfolio/assets/89055502/9e27d751-ab9b-47e9-b707-b65d227151b4)
- Your pop-up
![io](https://github.com/MargaretTechworld/Portfolio/assets/89055502/54e9daf5-29ff-4d7f-ad3c-20be1d4ccb8b)